examples/benchmarks/tablebench/.mocharc.customBenchmarks.cjs

```diff
@@ -13,19 +13,11 @@ const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mochar
 const packageDir = __dirname;
 const baseConfig = getFluidTestMochaConfig(packageDir);
 
-const nodeOptions =
-	baseConfig["node-option"] !== undefined
-		? Array.isArray(baseConfig["node-option"])
-			? baseConfig["node-option"]
-			: [baseConfig["node-option"]] // If string, wrap as array to use spread operator
-		: []; // If undefined, use an empty array
-
-nodeOptions.push("expose-gc", "gc-global", "unhandled-rejections=strict");
+baseConfig["node-option"].push("gc-global");
```

Comment:
everything else is already in the base config. I manually tested both the tests and benchmarks to ensure thy worked.